lib/OpenQA/IPC.pm

```diff
@@ -221,10 +221,16 @@ sub service {
 
 sub _dispatch {
     my ($self, $target, $command, @data) = @_;
-    my $object = $self->service($target);
-    log_debug("dispatching IPC $command to $target: " . pp(\@data));
-    my $ret = $object->$command(@data);
-    log_debug("IPC finished");
+    my $ret = 0;
```

Comment:
Maybe it's better not set a default for return here, it would make it `0` instead of `undef` if the code among L226-228 fails (thus changing the expected result from undef to 0)

Reply:
Ok, I'll change that.